app/messages.py

```diff
@@ -9,7 +9,9 @@
 FIELD_AVAILABLE_TO_MENTOR_IS_INVALID = {
     "message": "Field available_to_mentor" " is not valid."
 }
-
+USERNAME_FIELD_HAS_INVALID_LENGTH={
+    "message": "The name field has to be longer than 1 characters and shorter than 31 characters."
```

Comment:
there is a separate function that generates this message, but this message should be here indeed 🤔 , because of internationalization issues.
I think this is fine for now, but then we have to find a way to use the limit values into this message, without being here hardcoded. @Ritika5901234